python/ray/tests/test_numba.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+from numba import njit
+import pytest
+import numpy as np
+import unittest
+import ray
+
+
+@njit(fastmath=True)
+def centroid(x, y):
+    return ((x / x.sum()) * y).sum()
+
+
+# A Wrapper to call centroid func
+@ray.remote
+def centroid_wrapper(x, y):
+    return centroid(x, y)
+
+
+class NumbaTest(unittest.TestCase):
+    def setUp(self):
+        ray.init()
```

Comment:
```suggestion
        ray.init(num_cpus=1)
```

The reason for this is that we don't want Ray to start a lot of workers when this test is run on a large multi-core machine (because it may slow down the test).